content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -2,133 +2,150 @@
+   that no driver is selected for those printers.
 
-My Zebra Printer doesn't print anything
----------------------------------------
+The Zebra printer doesn't print anything
+----------------------------------------
 
-Zebra printers are quite sensitive to the format of the ZPL code that is
-printed. If nothing comes out of the printer or blank labels are printed,
-try changing the format of the report that is sent to the printer by
-accessing :menuselection:`Settings ~~> Technical ~~> Views` in developer
-mode and look for the corresponding template.
+Zebra printers are quite sensitive to the format of the ZPL code that is printed. If nothing comes
+out of the printer or blank labels are printed, try changing the format of the report that is sent
+to the printer by accessing :menuselection:`Settings --> Technical --> User Interface --> Views` in
+:ref:`developer mode <developer-mode>` and look for the corresponding template.
 
-Barcode Scanner
+Barcode scanner
 ===============
 
 The characters read by the barcode scanner don't match the barcode
 ------------------------------------------------------------------
 
-By default, we assume that your barcode scanner is configured in US
-QWERTY. This is the default configuration of most barcode readers.
-If your barcode scanner uses a different layout, please go to the
-form view of your device and select the correct one.
+By default, most barcode scanners are configured in the US QWERTY format. If the barcode scanner
+uses a different layout, go to the form view of the device (:menuselection:`IoT App --> Devices -->
+Barcode Device`) and select the correct format.
 
 Nothing happens when a barcode is scanned
 -----------------------------------------
 
-Make sure that the correct device is selected in your Point of Sale
-configuration and that your barcode is configured to send an ENTER
-character (keycode 28) at the end of every barcode.
+Make sure that the correct device is selected in the :menuselection:`Point of Sale` configuration
+and that the barcode is configured to send an ENTER character (keycode 28) at the end of every
+barcode. Navigate to :menuselection:`POS app --> 3-Dot Menu on the POS --> IoT Box section --> Edit
+--> Barcode Scanners`.
 
 The barcode scanner is detected as a keyboard
 ---------------------------------------------
 
-Some poorly built barcode scanners do not advertise themselves as
-barcode scanners but as a USB keyboard instead, and will not be
-recognized by the IoT Box.
+.. important::
+   Some barcode scanners do not advertise themselves as barcode scanners but as a USB keyboard
+   instead, and will not be recognized by the :abbr:`IoT (Internet of Things)` box.
 
-You can manually change the device type by going to its form view and
-activating the *Is scanner* option.
+The device type can be manually changed by going to its form view (:menuselection:`IoT App -->
```

Comment:
```suggestion
The device type can be manually changed by going to its form view (:menuselection:`IoT -->
```

Reply:
Again, saying `app` is fine in the `menuselection`